assets/src/scripts/contributors.js

```diff
@@ -0,0 +1,195 @@
+jQuery( function ( $ ) {
+	// Media
+	$( document ).ready( function () {
```

Comment:
I think we can avoid the double document.ready callback because the following is the shorthand 
`
jQuery( function ( $ ) { } 
`
Now we are nesting two document ready callbacks

Reply:
Probably it could be a good idea to cache in a variable some selectors like:

`jQuery( '#contributor-picture' )` to avoid querying the DOM every-time and also making it easy to refactor.

`const pictureField = jQuery( '#contributor-picture' )`

Then you can use it everywhere but the DOM lookup will only happen once.

`pictureField..val( '' )`